### PR TITLE
LPS-97666 buildLang

### PR DESCRIPTION
**Pull request by gabrielpramos**

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/process-metrics/filter/ProcessStepFilter.es.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ import Filter from '../../../shared/components/filter/Filter.es';
 import {ProcessStepContext} from './store/ProcessStepStore.es';
 
 const ProcessStepFilter = ({
+	classNames,
```
Comment (phcp) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/process-metrics/filter/ProcessStepFilter.es.js]: Rename to className

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/process-metrics/performance-by-assignee-card/PerformanceByAssigneeFilter.es.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+				<ProcessStepFilter
+					filterKey="assigneeProcessStep"
+					hideControl={true}
+					multiple={false}
+					position="right"
+					showFilterName={false}
+				/>
+
+				<TimeRangeFilter
+					className={'pl-3'}
```
Comment (phcp) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/process-metrics/performance-by-assignee-card/PerformanceByAssigneeFilter.es.js]: use as a simple string param

**Comment by gabrielpramos** [comment]
ci:test:sf

**Comment by gabrielpramos** [comment]
ci:test:relevant

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/process-metrics/ProcessMetrics.es.js [hunk 3]
```diff
@@ -182,7 +183,8 @@ class ProcessMetrics extends React.Component {
 							render={withParams(
 								CompletedItemsCard,
 								CompletionVelocityCard,
-								PerformanceByStepCard
+								PerformanceByStepCard,
+								PerformanceByAssigneeCard
```
Comment (phcp) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/process-metrics/ProcessMetrics.es.js:187]: SF

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-lang/src/main/resources/content/Language.properties [hunk 4]
```diff
@@ -106,6 +107,8 @@ pause=Pause
 pending-items=Pending Items
 pending-items-description=Pending workflow items are counted and categorized by their SLA status.
 performance=Performance
+performance-by-assignee=Performance by Assignee
```
Comment (phcp) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-lang/src/main/resources/content/Language.properties:110]: language changes should come in the same commit as its use

**Review by phcp — CHANGES_REQUESTED** [review]
Some initial changes, please check

**Pull request closed without merging**